pkgs/applications/networking/feedreaders/newsboat/default.nix

```diff
@@ -12,16 +12,24 @@ stdenv.mkDerivation rec {
 
   prePatch = ''
     substituteInPlace Makefile --replace "|| true" ""
+    # Allow other ncurses versions on Darwin
+    substituteInPlace config.sh \
+      --replace "ncurses5.4" "ncurses"
   '';
 
-  nativeBuildInputs = [ pkgconfig asciidoc docbook_xml_dtd_45 libxslt docbook_xml_xslt ]
+  nativeBuildInputs = [ pkgconfig asciidoc docbook_xml_dtd_45 libxslt docbook_xml_xslt libiconv ]
```

Comment:
This is only a dependency on darwin and should be move in the optional block a line below.

Reply:
Actually, `iconv.h` is referenced in [utils.cpp](https://github.com/NixOS/nixpkgs/pull/15647/commits/f3a753829d8e385255db2a85ff67798ea34c86fd) so it is a dependency on both, Darwin and Linux.
But maybe it's in one of the dependenies' `propagatedBuildInputs` for Linux only.

Reply:
Oh I see, `libiconv` is not needed on Linux because it is part of the standard library. Fixing it :)